### DISCUSSION TITLE
MTEX .gpol results

### DISCUSSION THREAD

**Discussion by savoiejean** [mtex-toolbox/mtex#1278] [Feature Requests]
Is it possible to have the pole figure data in ASCII format?

**Comment by kilir** [comment]
Hi,
'Polefigure.export()' will export into a text file. In my case the character set is ascii. 'Polefigure.load' also accepts ascii encoded files.

Cheers,
Rüdiger

**Comment by savoiejean** [comment]
It works. Thanks.
